post-scan-actions/aws-python-slack-notification/terraform/lambda.tf

```diff
@@ -0,0 +1,44 @@
+# Zip the lambda code
+data "archive_file" "zip_the_python_code" {
+  type        = "zip"
+  source_file = "../handler.py"
+  output_path = "${path.module}/lambda_zip/handler.zip"
+}
+
+# Create the lambda function
+resource "aws_lambda_function" "slack_notification_lambda" {
+  filename      = "${path.module}/lambda_zip/handler.zip"
+  function_name = "SlackNotificationLambda"
```

Comment:
I ran into the error of `Function already exist...` after trying to install the plugin again. I know the previous installation took the function name. But this might also happen to users with the same lambda name `SlackNotificationLambda` in their AWS account. Please consider using a uuid or a variable for `function_name`.

Reply:
Done using random!